changedetectionio/html_tools.py

```diff
@@ -97,18 +97,23 @@ def _parse_json(json_data, json_filter):
         match = jsonpath_expression.find(json_data)
         return _get_stripped_text_from_json_match(match)
 
-    if 'jq:' in json_filter:
+    if 'jq:' in json_filter or 'jqraw:' in json_filter:
 
         try:
             import jq
         except ModuleNotFoundError:
             # `jq` requires full compilation in windows and so isn't generally available
             raise Exception("jq not support not found")
 
-        jq_expression = jq.compile(json_filter.replace('jq:', ''))
-        match = jq_expression.input(json_data).all()
+        if 'jq:' in json_filter:
```

Comment:
Maybe `json_filter.startswith(...)`

Reply:
I would advise against utilizing `startswith`. It's more advisable to be exceptionally precise when it comes to use input.